.github/actions/setup-minikube-cluster/action.yml

```diff
@@ -41,6 +41,22 @@ runs:
             driver: '${{ inputs.minikube_driver }}'
             start args: '--container-runtime=${{ inputs.container_runtime }}'
 
+    - name: Create minikube Kubernetes ${{ inputs.k8s_version }} Cluster
+      id: create-minikube-cluster-124
+      # TODO: Workaround until conditions are supported natively
+      # See https://github.com/actions/runner/issues/834
```

Comment:
That issue has been resolved, and conditions are supported natively on steps:
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsif
https://github.blog/changelog/2021-11-09-github-actions-conditional-execution-of-steps-in-actions/
https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#runs-for-composite-actions